src/system/RoutesModule/AppSettings.php

```diff
@@ -1,17 +1,17 @@
 <?php
 
-declare(strict_types=1);
```

Comment:
in the rest of the core, the declaration is at the top (like this) not after the comment, like below. Why change it?

Reply:
Demanded by phpcs / PHP_CodeSniffer. Maybe to conform to PSR-12 standard.

Also it we are actually documenting the file, not the `namespace` keyword. Thus, the comment must come first. See also https://github.com/squizlabs/PHP_CodeSniffer/issues/1800#issuecomment-352634571

You don't need to mind about it. I will change rest of the core later. Refs #3987